yt_dlp/extractor/manoto.py

```diff
@@ -0,0 +1,102 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class ManotoTVIE(InfoExtractor):
+    IE_NAME = 'Manoto TV (Episode / Video)'
+    _VALID_URL = r'https?://(?:www\.)?manototv\.com/episode/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://www.manototv.com/episode/12576',
+        'info_dict': {
+            'id': '12576',
+            'title': 'Seh Mah Taatili',
+            'description': 'مجموعه ای از فیلم های سینمای کلاسیک ایران',
+            'thumbnail': r're:^https?://.*\.jpeg$',
+            'ext': 'm3u8',
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        json = self._download_json('https://dak1vd5vmi7x6.cloudfront.net/api/v1/publicrole/showmodule/episodedetails?id=' + video_id, video_id)
```

Comment:
don't call it `json`
Will cause confusion
rename it to `data_json` or `video_json` or `episode_json`

Reply:
Done.